microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/integration/FailSafeCDIHealthBaseTest.java

```diff
@@ -44,6 +46,8 @@ public void before() throws ConfigurationException, InterruptedException, Timeou
         String url = "http://" + arqProps.getDefaultManagementAddress() + ":" + arqProps.getDefaultManagementPort()
                 + "/metrics";
         metricsRequest = given().baseUri(url).accept(ContentType.JSON);
+        // reset MP Config properties
+        new Administration(ManagementClientProvider.onlineStandalone(managementClient)).reload();
```

Comment:
Client not closed.

Reply:
Actually, the client can not be closed. Doing so would also close `managementClient` managed by arquillian which is something we do not want to. I would not be able to deploy then since  `managementClient` would be closed.